package.json

```diff
@@ -21,7 +21,7 @@
       "av": {
         "slackChannel": "#av-api-alerts",
         "resources": [
-          "tests/bulk_intl_verifications_test.js",
+          "tests/intl_verifications_bulk_test.js",
           "tests/bulk_us_verifications_test.js",
```

Comment:
Why isn't this also changed to `us_verifications_bulk_test`?

Reply:
@zolopane and @sudoku-lord 

I'm seeing the changes to the Tag names removing whitespaces in the PR.  I don't think this should be merged.